config/secrets.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  jwtSecret:
-    process.env.JWT_SECRET ||
-    "lorem20lorem20lorem20lorem20lorem20lorem20lorem20lorem20lorem20lorem20"
+	jwtSecret:
+		process.env.JWT_SECRET ||
+		"lorem20lorem20lorem20lorem20lorem20lorem20lorem20lorem20lorem20lorem20",
```

Comment:
were these "space" changes on purpose or due to a formatting script/linter? If the latter please get on the same page as a team with format/linting quickly in order to limit wasteful diffs like this.